libminifi/test/TestBase.cpp

```diff
@@ -104,9 +104,9 @@ std::shared_ptr<core::Processor> TestPlan::addProcessor(const std::shared_ptr<co
     connection->setDestination(processor);
 
     utils::Identifier uuid_copy, uuid_copy_next;
-    last->getUUID(uuid_copy);
+    uuid_copy = last->getUUID();
     connection->setSourceUUID(uuid_copy);
-    processor->getUUID(uuid_copy_next);
+    uuid_copy_next = processor->getUUID();
     connection->setDestinationUUID(uuid_copy_next);
```

Comment:
minor, but this could be simplified to
```c++
connection->setSourceUUID(last->getUUID());
connection->setDestinationUUID(processor->getUUID());
```
(also further down in this file)

Reply:
done